From 95a358da7b015b252ad5177f215c14620358e109 Mon Sep 17 00:00:00 2001
From: johnmart19 <johnivan19999@gmail.com>
Date: Sat, 7 May 2022 22:41:03 +0300
Subject: [PATCH] Add Google Search and Google Location History to location
 indicator whitelist

* They are reported to be disturbing as well.
---
 .../src/com/android/systemui/privacy/PrivacyItemController.kt   | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/privacy/PrivacyItemController.kt b/packages/SystemUI/src/com/android/systemui/privacy/PrivacyItemController.kt
index a3ebe3603776..427c171a55ae 100644
--- a/packages/SystemUI/src/com/android/systemui/privacy/PrivacyItemController.kt
+++ b/packages/SystemUI/src/com/android/systemui/privacy/PrivacyItemController.kt
@@ -62,6 +62,8 @@ class PrivacyItemController @Inject constructor(
             "com.android.networkstack.tethering",
             "com.android.phone",
             "com.android.systemui",
+            "com.google.android.gms.location.history",
+            "com.google.android.googlequicksearchbox",
             "com.google.android.settings.intelligence",
             "com.mediatek.ims",
         )
